server/src/main/java/io/unitycatalog/server/service/AuthService.java

```diff
@@ -155,7 +158,27 @@ public HttpResponse grantToken(OAuthTokenExchangeRequest request) {
             .tokenType(AuthTypes.BEARER)
             .build();
 
-    return HttpResponse.ofJson(response);
+    // Set token as cookie if ext param is set to cookie
+    ResponseHeadersBuilder responseHeaders = ResponseHeaders.builder(HttpStatus.OK);
+    ext.ifPresent(
+        e -> {
+          if (e.equals(COOKIE)) {
+            // Set cookie timeout to 5 days by default if not present in server.properties
+            String cookieTimeout =
+                ServerProperties.getInstance().getProperty("server.cookie-timeout", "5D");
+            Cookie cookie =
+                Cookie.secureBuilder(AuthDecorator.UC_TOKEN_KEY, accessToken)
+                    .path("/")
+                    .maxAge(
+                        Duration.parse(
+                                "P" + (cookieTimeout.endsWith("D") ? "" : "T") + cookieTimeout)
```

Comment:
Lets just have the `server.properties` value be the full string the `parse()` method expects. Spring users for example, should be familiar with the full syntax.